src/Microsoft.DotNet.Build.Tasks/Targets/sign.targets

```diff
@@ -1,6 +1,14 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Project ToolsVersion="12.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
 
+  <!-- metadata used for Authenticode and strong-name signing in official VSO builds -->
+  <ItemGroup>
+    <FilesToSign Include="$(TargetPath)">
+      <Authenticode>Microsoft</Authenticode>
+      <StrongName Condition="'$(SkipSigning)' != 'true'">StrongName</StrongName>
```

Comment:
Given that you are redoing this should we change the condition to be more specific? $(SkipStrongNameSigning)? In case we need to have an opt out for the Authenticode as well.